libraries/AP_Baro/AP_Baro_DroneCAN.h

```diff
@@ -15,7 +15,7 @@ class AP_Baro_DroneCAN : public AP_Baro_Backend {
 
     void update() override;
 
-    static void subscribe_msgs(AP_DroneCAN* ap_dronecan);
+    static bool subscribe_msgs(AP_DroneCAN* ap_dronecan, uint8_t driver_index);
```

Comment:
could it be a reference?

Reply:
Technically possible, but doesn't save anything and means everything has to be `*this` and `&ap_dronecan`. I don't like it.